pkg/deploy/apis/apps/v1/conversion.go

```diff
@@ -119,10 +118,5 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 		return err
 	}
 
-	if err := scheme.AddFieldLabelConversionFunc("v1", "DeploymentConfig",
-		oapi.GetFieldLabelConversionFunc(newer.DeploymentConfigToSelectableFields(&newer.DeploymentConfig{}), nil),
-	); err != nil {
-		return err
-	}
```

Comment:
Why is this required [for buildconfigs](https://github.com/openshift/origin/pull/15963/files#diff-c6033988686cf6611cf7e928d8dbea2aR178) but not here?